cmd/minikube/main.go

```diff
@@ -205,7 +205,7 @@ func setFlags(parse bool) {
 		if pflag.CommandLine.Changed("log_dir") && pflag.Lookup("log_dir").Value.String() != "" {
 			dir = pflag.Lookup("log_dir").Value.String()
 		}
-		l := logFileName(dir, 0)
```

Comment:
`pflag` is a global variable, so you can avoid having to pass it as an argument to `logFileName`.